addons/calendar/tests/test_event_notifications.py

```diff
@@ -44,11 +45,29 @@ def test_message_invite_allday(self):
                 'partner_ids': [(4, self.partner.id)],
             }])
 
-
     def test_message_invite_self(self):
+        # The organizer is used as the author
+        self.event.user_id = self.user
+        with self.assertNoNotifications():
+            self.event.with_user(self.user_root).partner_ids = self.partner
```

Comment:
Isn't this one a bit strange?

I feel like if someone else assigns "you" as part of the attendees, you should always be notified, even if you're the organizer.
Did someone specifically ask for the organizer as author inside _send_mail_to_attendees?
Shouldn't the author remain "the person doing the change" in all cases?